### PR TITLE
Improve logs

### PR DESCRIPTION
**Pull request by andrecardoso**

### CODE REVIEW

**Comment by andrecardoso** [comment]
[(1) ChrOS: Improve marketwall display id logs](https://trello.com/c/whdnZtyo/4684-1-chros-improve-marketwall-display-id-logs)

File: package.json [hunk 1]
```diff
@@ -64,6 +64,6 @@
     ]
   },
   "engines": {
-    "node": ">=8.6.0"
+    "node": ">=10.0.0"
```
Comment (rudibravo) [inline package.json]: @andrecardoso just curious, why we were on node 8 and now updating to node 10?

Comment (andrecardoso) [reply package.json]: We are using `assert.rejects` and `assert.doesNotReject` that need node v10.0.0 🆕

Comment (rudibravo) [reply package.json]: @andrecardoso 👍

**Pull request merged**